hummingbot/connector/exchange/ftx/ftx_user_stream_tracker.py

```diff
@@ -22,16 +20,15 @@ def logger(cls) -> HummingbotLogger:
         return cls._btust_logger
 
     def __init__(
-        self,
-        ftx_auth: Optional[FtxAuth] = None,
-        trading_pairs: Optional[List[str]] = [],
+            self,
+            ftx_auth: Optional[FtxAuth] = None,
+            trading_pairs: Optional[List[str]] = [],
     ):
-        super().__init__()
         self._ftx_auth: FtxAuth = ftx_auth
+        super().__init__(data_source=FtxAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by `super().__init__`. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.